packages/strapi-plugin-content-manager/admin/src/assets/images/icon-cross-blue.svg

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
My bad... I forgot to minimize the SVG icons...

Reply:
Bad @Aurelsicoko bad :stuck_out_tongue: